src/vacuum-card.js

```diff
@@ -184,20 +186,52 @@ class VacuumCard extends LitElement {
       battery_level,
       battery_icon,
       friendly_name,
+      detected_pad,
+      tank_level,
     };
   }
 
+  renderMopSetting(source) {
+    const [mopBehavior, amount] = source.split('-').map((x) => x.trim());
+    const sprayAmount = {
+      '1': 'Low',
+      '2': 'Med',
+      '3': 'High',
+    }[amount];
```

Comment:
I think we should use here the same approach as for fan speeds. For example my water level is already a string and not a number. In addition I have 4 different levels, which would not work with your solution.